android/res/values/dimens.xml

```diff
@@ -141,6 +141,7 @@
   <dimen name="nav_frame_padding">@dimen/margin_half</dimen>
   <dimen name="zoom_buttons_margin">64dp</dimen>
   <dimen name="map_buttons_bottom_margin">136dp</dimen>
+  <dimen name="map_buttons_bottom_max_width">450dp</dimen>
```

Comment:
"I made the maximum width 300dp instead of 450dp "
So what is a final value?

Reply:
I did not push yet, I wanted to have feedback first